webapp/channels/src/components/with_tooltip/index.tsx

```diff
@@ -161,6 +161,8 @@ function WithTooltip({
         console.error('Children must be a valid React element for WithTooltip');
     }
 
+    const combinedFloatingStyles = Object.assign({}, floatingStyles, transitionStyles);
```

Comment:
should this go in a useMemo to avoid new object creating every time?

Reply:
Agreed will move